apps/remote_control/lib/lexical/remote_control.ex

```diff
@@ -9,7 +9,7 @@ defmodule Lexical.RemoteControl do
   alias Lexical.RemoteControl.ProjectNode
   require Logger
 
-  @allowed_apps ~w(common path_glob remote_control elixir_sense)a
+  @allowed_apps ~w(common path_glob remote_control elixir_sense sourceror)a
```

Comment:
We need to consider one issue: if we support plug-ins in the future, there may be many projects that depend on `sourceror`.

Reply:
We also need to solve dependency conflict issues.

Reply:
I'm not following, the other projects will need to figure out dependency issues with respect to sourceror, which shouldn't be too bad.